igor_src/main.cpp

```diff
@@ -958,6 +1022,20 @@ int main(int argc , char* argv[]){
 					seqs_fileformat = TXT_f;
 					clog<<"TXT extension detected for the input sequence file"<<endl;
 				}
+//				else if(tmp_str == ".fastq"){
+//					seqs_fileformat = FASTQ_f;
+//					clog<<"FASTQ extension detected for the input sequence file"<<endl;
+//					clog<<"Parsing fastq to fasta "<<endl;
+//					string basefilename = input_seqs_file.substr(0, extension_index);
+//					try{
+//					  system("fastaq_to_fasta -i "+input_seqs_file+" -o "+basefilename+".fasta");
```

Comment:
Good idea to support fastq files! However relying on an external program makes you go in circles (use another program to create a fasta from a fastq), why not simply create (or find/link against a library with) a fastq parser directly?

Reply:
Yeah, I was testing that and I change my mind that is why I comment that lines and I forget to delete it sorry for exactly the same reason.

Reply:
Could you cleanup and just add a /TODO Add support for FASTQ files, instead?